packages/graphql-transformers-e2e-tests/src/__tests__/AuthV2TransformerIAM.test.ts

```diff
@@ -0,0 +1,952 @@
+import { PrimaryKeyTransformer } from '@aws-amplify/graphql-index-transformer';
```

Comment:
This file is new test that covers multiple combinations of cognito and non-cognito access patterns.